hr_holidays_public/models/hr_employee.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Camptocamp - Telmo Santos
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    def list_work_time_per_day(
```

Comment:
the method is defined on resource.mixin, and this is the model you need to patch, otherwise all other models which inherit from the mixin will have the wrong method. 

See https://github.com/odoo/odoo/blob/14.0/addons/resource/models/resource_mixin.py#L167

Reply:
In my first implementation I did the override of the method `list_work_time_per_day` in resource.mixin. But as suggested by @i-vyshnevska this change should be applied only for employees. mixin is used in other objects and this behavior may not be wanted there

Reply:
then please add a comment in the source code explaining why you are doing this on that method and not on the mixin, and a unit test using another model inheriting from the mixin so that we get a clear information by reading the code that it is not an oversight.

Reply:
yep, it was my suggestion, https://github.com/OCA/hr-holidays/pull/36#discussion_r770707321 
work-center is not an employee why it should consider employee, as it OCA better to localize it just to employee

Reply:
@gurneyalex Regading the unit test where I should use a class inheriting from `resource.mixin` that is not `hr.employee`, I don't see how I can do such a test. For example `mrp.workcenter` is defined in mrp module and is not installed.

Reply:
@santostelmo  check the resource module tests: there is a ressource.test model you can use.